unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/AudioSource/ECSAudioSourceComponentHandler.cs

```diff
@@ -124,10 +135,16 @@ private void ApplyCurrentModel()
                     audioSource.Stop();
                 }
             }
+            else if(isAudioClipReady)
+            {
+                // PlayOneShot is faster ( about twice faster) than their Play counterpart
+                audioSource.PlayOneShot(audioClip);
```

Comment:
If I remember correctly when using PlayOneShot there is no possibility of Stopping the clip, it just stops when the clip is over, when applying the model there is a call to audioSource.Stop(), could this be problematic?

Reply:
You are right. I didn't take into account that we may want to stop the audio